plugins/devlink/fu-devlink-device.c

```diff
@@ -533,6 +533,20 @@ fu_devlink_device_setup(FuDevice *device, GError **error)
 		return FALSE;
 	}
 
+	/* use quirk database for a better name */
+	fu_device_add_instance_u16(device, "VEN", fu_device_get_vid(device));
+	fu_device_add_instance_u16(device, "DEV", fu_device_get_pid(device));
```

Comment:
This code needs to work also with devices without VID/PID, like netdevsim, i2c, etc....